packages/styleguide/src/components/Colors/ColorContext.tsx

```diff
@@ -175,20 +201,20 @@ export const ColorHtmlBodyColors = ({ colorSchemeKey = 'auto' }) => {
           colorSchemeKey === 'auto'
             ? [
                 // default light
-                `html, body { background-color: ${colors.light.default}; color: ${colors.light.text}; }`,
+                `html, body { background-color: ${colorsObject.light.default}; color: ${colorsObject.light.text}; }`,
                 // dark via user preference
-                `html[data-user-color-scheme="dark"], html[data-user-color-scheme="dark"] body { background-color: ${colors.dark.default}; color: ${colors.dark.text}; }`,
+                `html[data-user-color-scheme="dark"], html[data-user-color-scheme="dark"] body { background-color: ${colorsObject.dark.default}; color: ${colors.dark.text}; }`,
                 // os dark preference
                 `@media (prefers-color-scheme: dark) {`,
                 [
                   // auto dark via media query
-                  `html, body { background-color: ${colors.dark.default}; color: ${colors.dark.text}; }`,
+                  `html, body { background-color: ${colorsObject.dark.default}; color: ${colorsObject.dark.text}; }`,
                   // light via user preference when os is dark
-                  `html[data-user-color-scheme="light"], html[data-user-color-scheme="light"] body { background-color: ${colors.light.default}; color: ${colors.light.text}; }`,
+                  `html[data-user-color-scheme="light"], html[data-user-color-scheme="light"] body { background-color: ${colorsObject.light.default}; color: ${colors.light.text}; }`,
                 ].join('\n'),
                 `}`,
               ].join('\n')
-            : `html, body { background-color: ${colors[colorSchemeKey].default}; color: ${colors[colorSchemeKey].text}; }`,
+            : `html, body { background-color: ${colorsObject[colorSchemeKey].default}; color: ${colorsObject[colorSchemeKey].text}; }`,
       }}
     />
   )
```

Comment:
I suppose these changes will be fine, but be sure to deploy on staging and double-check if everything works as expected in normal articles (check light and dark-mode) and pages that force dark-mode (for-example the lesha articles).